src/batch-client/index.ts

```diff
@@ -603,13 +603,19 @@ export class ZimbraBatchClient {
 		}).then(Boolean);
 
 	public downloadAttachment = ({ id, part }: any) =>
-		this.download({ id, part }).then(({ id, part, content }: any) => ({
+		this.download({ url:`/service/home/~/?auth=co&id=${id}&part=${part}` }).then(({ id, part, content }: any) => ({
```

Comment:
callback in then expects id, part will be passed by download function but we have removed it

Reply:
Done